src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/SettingsComponentTest.kt

```diff
@@ -210,7 +235,12 @@ class SettingsComponentTest {
 
         // Open settings again
         val ideaFrame = find(IdeaFrame::class.java, timeout = Duration.ofSeconds(15))
-        ideaFrame.openSettings()
```

Comment:
Maybe you can have your change of `ideaFrame.openSettings()` inside the method `openSettings()` to reduce logic duplication?

Reply:
Will try

Reply:
I think I resolved your comment. If that's what you wanted. However, I ran into a flaky test that I am trying to fix now. 😅